Makefile

```diff
@@ -21,7 +21,7 @@ clean:
 	go clean
 
 test:
-	go test -race -v ./...
+	CGO_ENABLED=1 go test -race -v ./...
```

Comment:
This seems to be required when running on `ubuntu:22.04` vs on the host `ubuntu-latest` runner.